src/utils/paymentMethod.js

```diff
@@ -0,0 +1,3 @@
+const hasPaymentMethod=(paymentMethod) => paymentMethod.id;
```

Comment:
Check if BuyCrypto is migrated to use this method. Right now it's still using selectWyrePaymentMethod and it's checking the existence of the object.